### ISSUE TITLE
Customizing shortcuts

### ISSUE THREAD

**Issue by CaptainDario** [pmutisya/keymap#8]
Hello,
this is a great package. However, I am wondering if there is a way to create shortcuts from strings, so that a user can define his/her own shortcuts.

Smth. like
``` dart
KeyboardWidget(
 bindings: [
  KeyAction(LogicalKeyboardKey("A"), "Triggered when a is pressed", () => Clear(strokes)
...
)
```

**Comment by pmutisya** [comment]
Excellent suggestion. Implemented in version 0.0.91

**Comment by CaptainDario** [comment]
@pmutisya thanks!
does this allow for multiple keys? `Like Shift + A + B` triggers something?
